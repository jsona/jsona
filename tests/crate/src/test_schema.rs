snapshot_schema_parse!("schema.jsona", parse_spec);
snapshot_schema_point!(
    "schema.jsona",
    point_spec,
    "_.value.bool",
    "_.value.array",
    "_.value.object2.k1",
    "null.value",
    "object.value.k2"
);
