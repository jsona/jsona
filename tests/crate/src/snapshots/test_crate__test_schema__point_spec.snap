---
source: tests/crate/src/test_schema.rs
expression: "serde_json::to_string_pretty(&output).unwrap()"
---
{
  "_.value.bool": [
    {
      "type": "boolean",
      "default": true
    }
  ],
  "_.value.array": [
    {
      "type": "integer"
    },
    {
      "type": "string"
    }
  ],
  "_.value.object2.k1": [
    {
      "type": "string"
    }
  ],
  "null.value": [
    {
      "type": "null"
    }
  ],
  "object.value.k2": [
    {
      "type": "string"
    }
  ]
}
